source/fundamentals/crud/write-operations/pkFactory.txt

```diff
@@ -0,0 +1,99 @@
+.. _node-pkfactory:
+
+===================
+Primary Key Factory
```

Comment:
issue: I like this title for the page in general, but the way it shows up in the TOC doesn't allow it to work. I think to make it fit with the rest of the TOC items, it should be changed to something more action-oriented like "Generate your own _id"